pom.xml

```diff
@@ -100,6 +100,14 @@
     </dependencyManagement>
 
     <dependencies>
+
```

Comment:
This is not part of the description of the PR. Stray code change ? if not, please add it to the description